grommet/grommet
# Fix: rel attribute added to Button

#### What does this PR do?
Aims to resolve #5537 by adding the `rel` attribute (in the anchor tag context) to Button.

#### Where should the reviewer start?
`Button/index.d.ts` to view the `RelType` list and see how it was added to Button

#### What testing has been done on this PR?
Manual testing by adding `rel` to a `Button` component to make sure TypeScript no longer complains about the attribute

#### How should this be manually tested?
Add a `rel` attribute to any `Button` as you would an anchor tag and take notice that TypeScript does not give a warning / error

#### Any background context you want to provide?

#### What are the relevant issues?
#5537
#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
Yes, will be in its own PR under the grommet-site repo

#### Should this PR be mentioned in the release notes?
No need

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
